# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../mc/0003-mc-4.1.40-pre9-include-fs.h.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Fix to get 'FS_DIRSYNC_FL', 'FS_TOPDIR_FL', 'FS_NODUMP_FL', 'FS_NOATIME_FL' declared

--- mc-4.1.40-pre9/vfs/undelfs.c.orig	2011-04-12 17:03:42.896076255 +0200
+++ mc-4.1.40-pre9/vfs/undelfs.c	2011-04-12 17:04:29.652071975 +0200
@@ -47,6 +47,7 @@
 #include <stdlib.h>
 /* asm/types.h defines its own umode_t :-( */
 #undef umode_t
+#include <linux/fs.h>
 #include <linux/ext2_fs.h>
 #include <ext2fs/ext2fs.h>
 #include <ctype.h>
